README.md

```diff
@@ -60,6 +60,7 @@ Then, `kubectl apply -f deploy/mm.yaml`.
     * "use-spot": This will make the minion-manager intelligently use spot instances in the ASG
     * "no-spot": This will make the minion-manager always use on-demand instances in the ASG. This is useful when someone wants to temporarily switch to on-demand instances and at a later point switch to "use-spot"
     * Note that after changing the tag value, it may take upto 5 minutes for the minion-manager pod to see the changes and make them take effect.
+* The "k8s-minion-manager/not-terminate" tag can control ASG instance terminate by the minion-manager. If you want to control when to terminate ASG instances. You can set this tag to `true`. If not set or other value will disable this feature.
```

Comment:
Is this tag for the ASG or for specific instances in the ASG? If the tag is for the ASG itself, better option would be to simply set the value of the `k8s-minion-manager` tag to `no-spot`. That way, the instances in the ASG will use on-demand instances and never switch (therefore, never terminate) instances.

Reply:
I want to have chance to keep an instances running instead of auto terminated. When I use `use-spot`. And there have change MM changed launch config to on-demand. Then switch back to spot.

Reply:
So, the tag is for each instance, correct?

Reply:
No, it is set on ASG.
If change the `k8s-minion-manager` from `use-spot` to `no-spot`. Then MM will terminate all spot instance one by one. And vice versa.
So that I want to add a tag on ASG. Let MM know if this tag exist. Don't terminate instance for this ASG.
When that tag is removed or value changed. MM will handle that ASG instance termination.